src/compiler/CplaceTypescriptCompiler.ts

```diff
@@ -109,7 +109,10 @@ export class CplaceTypescriptCompiler extends AbstractTypescriptCompiler {
                 libraryExport: 'default'
             },
             plugins: [
-                new CReplacePlugin()
+                new CReplacePlugin(),
+                new webpack.IgnorePlugin({
```

Comment:
@vladimir-arsov This could might be excluded in the tsconfig.json -> `CplaceTSConfigGenerator.ts`

Reply:
The problem is that when webpack bundles the application code (with`app.js` entry), for some reason it picks up the index.js file also. I don't know why this happens, since this file is not required by the application code. So it's not related to the typescript compiler and tsconfig, but to the webpack.